src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -948,6 +948,92 @@ It is advised to embed credentials and certificates in the configuation file, in
 
 - *image:* (Optional) Docker image used to deploy ProActive Nodes. When it is not set, its default value `java:8-jre` is used.
 
+- *nodeDeploymentTemplateFile:* (Optional) Kubernetes deployment template file for deploying ProActive node pods. This template file tells Kubernetes platform the desired running environment for the ProActive node. Each ProActive node will use this template file to create its specific Kubernetes deployment. 
+The default deployment template file creates a running environment with both ProActive node and a Docker server, so that the tasks running on the Kubernetes nodes can execute Docker commands. It is supported through the Docker-in-Docker approach (i.e., running a Docker daemon inside the Kubernetes host container system). Specifically, it creates one pod with two containers (`proactive-node` and `dind-daemon`). 
+The template file can use the following macros, which will be interpreted and filled in the corresponding parameter values by Kubernetes Infrastructure later.
```

Comment:
The template file can use the following macros, which will be interpreted and filled **with** the corresponding parameter values by Kubernetes Infrastructure later.